include/webserv.hpp

```diff
@@ -12,8 +12,13 @@
 #include <algorithm>
 #include <sys/socket.h>
 #include <netdb.h>
+#include <cstdio>
 #include "Debug.hpp"
 
+#define SEND_BUFFER_SIZE 1024 * 100 // 100 KB
+const int BUFFER_SIZE = 1025;
```

Comment:
Why do we need a global variable?

Reply:
We had it already, I just moved it from wherever it was, because of problem of dependencies. But I see what you mean now. I'll tranform it in a macro.

Reply:
Ok. This should be solved now.